app/views/users/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_for :user, url: users_path do |f| %>
```

Comment:
You can pass an actual model to the form builder and it will figure out which url to use based on the model you pass it 

`form_for @user do |f|`